src/Compilers/Core/Portable/Operations/ISingleVariableDeclaration.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Semantics
+{
+    /// <summary>
+    /// Represents a single variable declaration and initializer.
+    /// </summary>
+    /// <para>
+    /// Current Usage:
+    ///   (1) C# variable declarator
+    ///   (2) VB single variable declaration
+    /// </para>
```

Comment:
Catch is just a variable declaration, right? I assumed that this would be sufficient.

---
In reply to: [146940846](https://github.com/dotnet/roslyn/pull/22842#discussion_r146940846) [](ancestors = 146940846)

Reply:
Added VB and C# catch declarations.

---
In reply to: [146977029](https://github.com/dotnet/roslyn/pull/22842#discussion_r146977029) [](ancestors = 146977029,146940846)